client/src/components/AlumniMentorship.js

```diff
@@ -641,6 +693,45 @@ class RequestCards extends Component {
                     </Modal.Actions>
                 </Modal>
             }
+            {this.state.showActionItemsModal && 
+                <Modal open={this.state.showActionItemsModal}>
+                    <Modal.Header>Add action items for {this.state.requestDetails.student.name}</Modal.Header>
+                    <Modal.Content>
+                    <Grid stackable>
+                        <Grid.Row columns={"equal"}>
+                            <Grid.Column width={4}>
+                                <Image
+                                    floated='left'
+                                    size='small'
+                                    src={this.state.requestDetails.student.imageURL}
+                                    rounded
+                                />
+                            </Grid.Column>
+                            <Grid.Column>
+                                <Modal.Description> 
+                                    Please select up to 3 action items for {this.state.requestDetails.student.name} 
+                                </Modal.Description>
+                                <PooledMultiSelectDropdown 
+                                 allowAddition='true'
+                                 endpoint={`/drop/actionItems`}
+                                 dataType={"Action Item"}
+                                 getInputs={this.getActionItemsInput.bind(this)}
+                                />
+                            </Grid.Column>
+                        </Grid.Row>
+                    </Grid>
+                </Modal.Content>
+                    <Modal.Actions>
+                        <Button onClick={this.toggleActionItemModal.bind(this)}>
+                            Cancel
+                        </Button>
+                        <Button onClick={this.submitActionItems.bind(this)} 
+                         disabled = {this.state.existingActionItems || this.state.newActionItems} primary>
```

Comment:
So the disabled conditional here should be `!(this.state.existingActionItems && this.state.existingActionItems.length) && !(this.state.newActionItems && this.state.newActionItems.length)` Let me know if that makes sense!
Also nit: let indent on `disabled` and have primary be on a new line